riak/tests/test_riak.py

```diff
@@ -421,13 +358,13 @@ def test_check(aggregator, spin_up_riak):
     aggregator.all_metrics_asserted()
 
 
-def test_bad_config(aggregator, spin_up_riak):
-    riak_check = Riak(CHECK_NAME, {}, {})
+@pytest.mark.usefixtures('dd_environment')
+def test_bad_config(aggregator, check):
     with pytest.raises(socket.error):
-        riak_check.check({"url": "http://localhost:5985"})
+        check.check({"url": "http://localhost:5985"})
 
     sc_tags = ['url:http://localhost:5985']
-    for sc in aggregator.service_checks(SERVICE_CHECK_NAME):
+    for sc in aggregator.service_checks(common.SERVICE_CHECK_NAME):
```

Comment:
Could this just be:

`aggregator.assert_service_check(common.SERVICE_CHECK_NAME, status=Riak.CRITICAL, tags=sc_tags)`